Modules/Microsoft365DSC/DSCResources/MSFT_EXOInboundConnector/readme.md

```diff
@@ -2,4 +2,4 @@
 
 ## Description
 
-Create a new Inbound connector in your cloud-based organization.
+This resource configures a Inbound connector in your cloud-based organization.
```

Comment:
Picky, but this goes on the official website. It should say:

[...]configures **an** Inbound connector [...]

Reply:
Fixed